requirements/default.txt

```diff
@@ -1,5 +1,4 @@
-celery>=4.3.0,<5.0.0; python_version>="3.7"
-vine==1.3.0
+celery>=5.0.0; python_version>="3.7"
```

Comment:
I'm not sure what changed since and if I added it.
Celery is tested against 3.6+ (https://github.com/celery/celery/blob/master/tox.ini#L3)
I'm not sure if the python version restriction will be deferred from Celery's requirements anyway.
Please delete it if you think it's redundant

Reply:
Done.

Reply:
celery supports python 3.6 and onwards

Reply:
@auvipy I already removed that marker.